app/models/miq_template.rb

```diff
@@ -1,5 +1,6 @@
 class MiqTemplate < VmOrTemplate
   include CustomActionsMixin
+  include EmsRefreshMixin
```

Comment:
This is already included in the base `VmOrTemplate` class so this shouldn't be necessary